### ISSUE TITLE
 wild voidpets disappear when I try to catch them

### ISSUE THREAD

**Issue by Grievingwidow** [voidpet/bugs#676]
When i try to catch wild voidpets they just appear somewhere else on the screen after i tap them and then disappear again completely and don't reappear again .

This has happened several times now !

**Comment by CosmicSymbols** [comment]
Yeah same! It's super frustrating with how time consuming and annoying the new catching mechanic is. Especially when the pets are constantly moving across the screen in addition to the rotating quick-time event since it can take forever to avoid wasting like 20 containers per pet. It sucks all the fun out of capturing when the good 5-6 minutes you just spent trying to get this one pet goes to waste because it dips out of existence.

**Comment by Hinomori-Shiho** [comment]
it just happens ig
I _believe_ that the force capture will be returned but I think the new mechanic is here to stay
